From 2eff1510fcf396db976dfb7078b47818c90301b0 Mon Sep 17 00:00:00 2001
From: Andreas Mohr <and@gmx.li>
Date: Sat, 28 May 2016 09:40:49 +0000
Subject: [PATCH] Cleanup compile warning on older gcc compilers

Cleanup compile error on older gcc compilers.

search.c: In function 'mc_search_set_error':
search.c:497:36: error: declaration of 'mc_search' shadows a global declaration [-Werror=shadow]
search.c:419:1: error: shadowed declaration is here [-Werror=shadow]

Signed-off-by: Andreas Mohr <and@gmx.li>
---
 lib/search/search.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/search/search.c b/lib/search/search.c
index af33b73..7190fa9 100644
--- a/lib/search/search.c
+++ b/lib/search/search.c
@@ -494,18 +494,18 @@ mc_search_getend_result_by_num (mc_search_t * lc_mc_search, int lc_index)
  */
 
 void
-mc_search_set_error (mc_search_t * mc_search, mc_search_error_t code, const gchar * format, ...)
+mc_search_set_error (mc_search_t * lc_mc_search, mc_search_error_t code, const gchar * format, ...)
 {
-    mc_search->error = code;
+    lc_mc_search->error = code;
 
-    MC_PTR_FREE (mc_search->error_str);
+    MC_PTR_FREE (lc_mc_search->error_str);
 
     if (format != NULL)
     {
         va_list args;
 
         va_start (args, format);
-        mc_search->error_str = g_strdup_vprintf (format, args);
+        lc_mc_search->error_str = g_strdup_vprintf (format, args);
         va_end (args);
     }
 }
